index.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+var semver = require('semver');
+var execa = require('execa');
+var del = require('del');
+
+var exec = function (cmd, args) {
+	const result = execa.sync(cmd, args);
+
+	if (result.stderr !== '') {
```

Comment:
Is it possible to do a new `execa` release or should I use `console.log/err` for now?